[ResponseOps][TaskManager] followups from resource based scheduling PR (#192124)
elastic/kibana: docs/user/production-considerations/task-manager-production-considerations.asciidoc

Other times it, might be preferable to increase the throughput of individual {kib} instances.

> Tweak the *Capacity* via the <<task-manager-settings,`xpack.task_manager.capacity`>> setting, which allows each {kib} to pull a higher number of tasks per interval. This could impact the performance of each {kib} instance as the workload will be higher.

Reviewer: It is unclear to me why "Capacity" and "Poll interval" are bolded here, since that typically implies they are UI elements but no UI is mentioned here. If you agree that it's not a UI element, lmk and I can update the paragraph that follow this one too.

```suggestion
Tweak the capacity with the <<task-manager-settings,`xpack.task_manager.capacity`>> setting, which enables each {kib} instance to pull a higher number of tasks per interval. This setting can impact the performance of each instance as the workload will be higher.
```

Author: Yes I agree, I don't think these are UI elements

Reviewer: I've added 7f7c3d3d97ef8b6f89e2528c179fe95bc2703a7d to make similar changes in the subsequent paragraph.
